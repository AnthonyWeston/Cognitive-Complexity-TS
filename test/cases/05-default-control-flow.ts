function f() {
    throw "";
}

function g() {
    return;
}

function h() {
    while (true) {
        continue;
    }
}

function i() {
    while (true) {
        break;
    }
}
