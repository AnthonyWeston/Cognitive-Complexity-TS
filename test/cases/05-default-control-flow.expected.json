{
    "05-default-control-flow.ts": {
        "score": 2,
        "inner": [
            {
                "name": "f",
                "line": 1,
                "column": 1,
                "score": 0
            },
            {
                "name": "g",
                "line": 5,
                "column": 1,
                "score": 0
            },
            {
                "name": "h",
                "line": 9,
                "column": 1,
                "score": 1
            },
            {
                "name": "i",
                "line": 15,
                "column": 1,
                "score": 1
            }
        ]
    }
}